package work;

public class Ex03 {

	public static void main(String[] args) {
		
		System.out.println("throws에 대한 설명으로 틀린 것은 무엇입니까?");
		
		System.out.println("1)생성자나 메소드의 선언 끝 부분에 사용되어 내부에서 발생된 예외를 떠넘긴다.");
		System.out.println("2)throws뒤에는 떠넘겨야 할 예외를 쉼표(,)로 구분해서 기술한다.");
		System.out.println("3)모든 예외를 떠넘기기 위해 간단하게 throws Exception으로 작성할 수 있다.");
		System.out.println("4)새로운 예외를 발생시키기 위해 사용된다.   -  x");

	}

}
